prebid-video/video-overview.md

```diff
@@ -81,6 +83,32 @@ The ad server chooses the winning line item. If a Prebid line item is selected,
 5.	**Prebid renders the outstream video.**
 Prebid retrieves the winning outstream video bid and renderer from the ad ID (hb_adid).  Prebid renders the outstream video into the banner slot.
 
+### Long-form Video 
+
+1.	**Prebid sends request to demand partners.**
+Prebid.js code loads within the page header and sends a bid request to each video demand partner included on a given Prebid video ad unit.
+
+2.	**Demand partners respond.**
+As with instream videos, each response includes the bid price and the video creative in the form of a VAST tag URL which returns a VAST XML wrapper.  This video creative will be rendered by the video player if the bid is selected in the ad server.  
+ 
+      Additionally each response is required to return an IAB subcategory. A module is provided that converts adserver categories to IAB sub categories. If a publisher prefers to use their own mapping file they will need to set the URL location of that file.  
+
+3.	**Prebid.js caches video bids.**  
+Each video bid is cached server-side and mapped to a unique cache ID which will be passed to the Freewheel ad server via key-value targeting.  The [Prebid.js video creative]({{site.github.url}}/adops/setting-up-prebid-video-in-dfp.html#creative-setup) configured in the ad server contains a macro that references this cache ID. (Note: Some bidders will cache before responding to Prebid.js. In those cases, the bidder will provide a `bidResponse.videoCacheKey` and this step will be skipped. See [Notes on Prebid Cache]({{site.github.url}}/dev-docs/show-video-with-a-dfp-video-tag.html#notes-on-prebid-cache) for details.) 
+
+4.	**Prebid creates new master video ad server tag URL.**  
+Prebid combines an existing video ad server tag with Prebid key-value targeting pairs to create a new master video ad server tag URL.  This URL will be passed into the video player.
+
+5.	**Video player calls ad server.**  
+The video player loads the master video ad server tag URL, which makes a call to the ad server.
+
+6.	**Ad server matches Prebid.js key-value pairs to a pre-configured line item.**
+
+7.	**Video player renders the video from the winning bidder.**  
+  a.	The master video ad server tag URL returns a VAST XML wrapper containing the Prebid.js video creative.
+  b.	The Prebid.js video creative returns a VAST XML document containing the cached video bid, which was returned by the Prebid demand partner in its bid response.
+  c.	The demand partner’s VAST tag URL is rendered in the video player.
```

Comment:
The a, b,c listing structure here isn't properly formatted as a list on the rendered site for me.  It's appearing as a giant paragraph.

See below:
![abc_formatting](https://user-images.githubusercontent.com/31102355/53179068-062bba00-35c1-11e9-81e7-403b76690a64.png)